### ISSUE TITLE
OrbitControls events don't fire if a camera is explicitly included in the scene

### ISSUE THREAD

**Issue by gpoole** [pmndrs/drei#895]
- `three` version: 0.140.2
- `@react-three/fiber` version: 8.0.13
- `@react-three/drei` version: 9.7.1
- `node` version: 16.14.0
- `npm` (or `yarn`) version: 1.22.17

### Problem description:

When a `PerspectiveCamera` is present in the scene, the `OrbitControls` event callbacks (`onStart`, `onEnd` and `onChange`) don't fire. If the camera is not present they fire normally. The actual controls work fine whether the camera is present or not, it's just the events that don't work.

One extra detail is that I've found that when I cause a hot reload in Next.js by saving a change, the events start firing normally even with the `PerspectiveCamera` present, so might be related to order of initialisation?

Could be related to #477 but I'm not sure.

### Relevant code:

This code demonstrates how including the camera causes the issue:

```jsx
const PerspectiveCameraScene = () => (
  <Canvas>
    {/* Uncomment the pespective camera, reload, and check console to demonstrate the problem */}
    {/* <PerspectiveCamera makeDefault position={[0, 0, -10]} /> */}
    <Box position={[0, 0, 0]} />
    <OrbitControls
      onStart={() => {
        console.log("start");
      }}
      onEnd={() => {
        console.log("end");
      }}
      onChange={() => {
        console.log("change");
      }}
    />
  </Canvas>
);
```

Also see this Sandbox: https://codesandbox.io/s/magical-cdn-5m5sux?file=/src/App.js

**Comment by felix-gohla** [comment]
Probably a duplicate of #873? You could check whether [`v9.11.3`](https://github.com/pmndrs/drei/releases/tag/v9.11.3) fixed it.

**Comment by gpoole** [comment]
Yes looks like it is @felix-gohla, can reproduce the issue in 9.11.2 but fixed in 9.11.3. Thanks!
